appveyor.yml

```diff
@@ -41,6 +47,7 @@ build_script:
 # because otherwise certain things (like python; import numpy) do not work
 
 cache:
-# cache toolchain installation file to avoid downloading it from AWS S3 each time
-# it's ~496MB < 1GB free limit for build caches
-- C:\Toolchain.msi -> appveyor.yml
+# cache the entire toolchain installation folder to avoid
+# downloading it from AWS S3 and installing the MSI each time
+# it's ~1.8GB > 1GB free limit for build caches
+- C:\PX4 -> appveyor.yml
```

Comment:
No, if you build locally with your toolchain the checkout is by default into the `C:\PX4\home` directory. But in CI `C:\PX4` only contains the toolchain and I use the default path of appveyor:

> git clone -q https://github.com/PX4/Firmware.git **C:\projects\firmware**

The build is ran in the backed up $repopath which is the path appveyor releases me for building:

> for /f %%i in ('cygpath -u %%CD%%') do set repopath=%%i

It doesn't make sense to separate the toolchain (note lower-case) folder because the configuration files around it are worst case ~200KB.